### DISCUSSION TITLE
Failure at working copy checkout only on Win 10

### DISCUSSION THREAD

**Discussion by NixToDo** [community/community#39842] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi all,
If I clone this (https://github.com/johnwinans/2063-Z80-cpm.git) github repro under Windows 10 with TortoiseGit or any other (CLI) tool it will clone it, but the creation of the working copy failed (folder is empty) with following message:

git.exe clone --progress -v "https://github.com/johnwinans/2063-Z80-cpm.git" "D:\Z80 Retro SBC\2063-Z80-cpm"
Cloning into 'D:\Z80 Retro SBC\2063-Z80-cpm'...
POST git-upload-pack (185 bytes)
POST git-upload-pack (gzip 1162 to 617 bytes)
remote: Enumerating objects: 631, done.
remote: Counting objects: 100% (202/202), done.
remote: Compressing objects: 100% (130/130), done.
remote: Total 631 (delta 106), reused 148 (delta 64), pack-reused 429
Receiving objects: 100% (631/631), 488.11 KiB | 1.90 MiB/s, done.
Resolving deltas: 100% (353/353), done.
error: invalid path 'lib/prn.asm'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

git did not exit cleanly (exit code 128) (3172 ms @ 23.11.2022 14:02:14)

git status shows me:
On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        deleted:    .gitignore
        deleted:    LICENSE
        deleted:    Makefile
        deleted:    README-SD.md
        deleted:    README.md
        deleted:    boot/Makefile
        deleted:    boot/README.md
        deleted:    boot/firmware.asm
        deleted:    cpm22/.gitignore
        deleted:    cpm22/Makefile
        deleted:    cpm22/README.md
        deleted:    doc/.gitignore
        deleted:    doc/2063-Z80-cpm.tex
        deleted:    doc/Make.rules
        deleted:    doc/Makefile
        deleted:    doc/README.md
        deleted:    doc/castellatedboxes.tex
        deleted:    doc/retro.pdf
        deleted:    filesystem/.gitignore
        deleted:    filesystem/Makefile
        deleted:    filesystem/README.md
        deleted:    filesystem/diskdefs
        deleted:    filesystem/progs/README.md
        deleted:    filesystem/progs/basic/README.md
        deleted:    filesystem/progs/basic/fighter.bas
        deleted:    filesystem/progs/example/.gitignore
        deleted:    filesystem/progs/example/Makefile
        deleted:    filesystem/progs/example/README
        deleted:    filesystem/progs/example/test1.asm
        deleted:    filesystem/progs/example/test2.asm
        deleted:    filesystem/progs/example/test3.asm
        deleted:    filesystem/progs/tms9118/Makefile
        deleted:    filesystem/progs/tms9118/README.md
        deleted:    filesystem/progs/tms9118/dram.asm
        deleted:    filesystem/progs/tms9118/green.asm
        deleted:    filesystem/progs/tms9118/mode1.asm
        deleted:    filesystem/progs/tms9118/mode1i2.asm
        deleted:    filesystem/progs/tms9118/mode2.asm
        deleted:    filesystem/progs/tms9118/white.asm
        deleted:    filesystem/progs/tty/.gitignore
        deleted:    filesystem/progs/tty/Makefile
        deleted:    filesystem/progs/tty/tty.asm
        deleted:    hello/Makefile
        deleted:    hello/README.md
        deleted:    hello/hello.asm
        deleted:    lib/ctc1.asm
        deleted:    lib/ctc2.asm
        deleted:    lib/ctc3.asm
        deleted:    lib/hexdump.asm
        deleted:    lib/io.asm
        deleted:    lib/memory.asm
        deleted:    lib/prn.asm
        deleted:    lib/puts.asm
        deleted:    lib/sdcard.asm
        deleted:    lib/sio.asm
        deleted:    lib/spi.asm
        deleted:    retro/.gitignore
        deleted:    retro/Makefile
        deleted:    retro/README.md
        deleted:    retro/retro.asm
        deleted:    retro/rw_dmcache.asm
        deleted:    retro/rw_nocache.asm
        deleted:    retro/rw_stub.asm
        deleted:    tests/Makefile
        deleted:    tests/README.md
        deleted:    tests/sd_test.asm
        deleted:    tests/spi_test.asm

git restore --source=HEAD :/ returns with:
error: invalid path 'lib/prn.asm'

I fork the repro with the github webpage interface and then tried to clone my forked repro. It fails too.
Same with the github GUI tool (Windors program). 

On a Linux system (I used a Raspberry Pi) all went well.
Do anyone know how / where the failure could be corrected?
Thanks Dirk!

**Comment by ossanna16** [comment]
Hi there @NixToDo and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
